### PR TITLE
Create profile model, T1S3

### PR DESCRIPTION
**Pull request by CrazzyBoie**

### CODE REVIEW

File: Models/Profile.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    class Profile
+    {
+        [DataMember]
+        public int ID { get; set; }
+        [DataMember]
+        public string FirstName { get; set; }
+        [DataMember]
+        public string LastName { get; set; }
+        [DataMember]
+        public string Description { get; set; }
```
Comment (EBazela) [inline Models/Profile.cs]: string is a primitive, don't use String class :)

File: ClientMVC/Models/Profile.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        [DataMember]
+        public int UserID { get; set; }
+        [DataMember]
+        public string FirstName { get; set; }
+        [DataMember]
+        public string LastName { get; set; }
+        [DataMember]
+        public String Description { get; set; }
+        [DataMember]
+        public Nullable<decimal> Longitude { get; set; }
```
Comment (EBazela) [inline ClientMVC/Models/Profile.cs]: It is nullable here

File: Models/Profile.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    class Profile
+    {
+        [DataMember]
+        public int ID { get; set; }
+        [DataMember]
+        public string FirstName { get; set; }
+        [DataMember]
+        public string LastName { get; set; }
+        [DataMember]
+        public String Description { get; set; }
```
Comment (EBazela) [inline Models/Profile.cs]: lowcase String please

File: ClientMVC/Models/Profile.cs [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        [DataMember]
+        public int ID { get; set; }
+        [DataMember]
+        public int UserID { get; set; }
+        [DataMember]
+        public string FirstName { get; set; }
+        [DataMember]
+        public string LastName { get; set; }
+        [DataMember]
+        public String Description { get; set; }
```
Comment (ciprianprohozescu) [inline ClientMVC/Models/Profile.cs]: Use primitive "string" (not "String").

File: ClientMVC/Models/Profile.cs [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        [DataMember]
+        public Nullable<decimal> Latitude { get; set; }
+        [DataMember]
+        public string ProfilePicture { get; set; }
+        [DataMember]
+        public Nullable<System.DateTime> Deleted { get; set; }
+
+        public Profile(int id, string firstName, string lastName, string description, decimal longitude, decimal latitude, string profilePicture)
+        {
+            this.ID = id;
```
Comment (EBazela) [inline ClientMVC/Models/Profile.cs]: fafjabfjfaj

**Pull request merged**